### PR TITLE
Adding hometown file for kousi2

### PR DESCRIPTION
**Pull request by kousi2**
@akorecky What kind of restaurants do you like so I can add them to the hometown file?

### CODE REVIEW

File: kousi2-chicago.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Chicago, IL
+
+### Resturants
+- Greek Islands
+
```
Comment (akorecky) [inline kousi2-chicago.md]: I really like :pizza:  and :green_salad:s.

**Review by akorecky — CHANGES_REQUESTED** [review]
Added some type of foods I like.

**Comment by kousi2** [comment]
@akorecky Added a couple nice places, let me know if this is good.

**Review by akorecky — APPROVED** [review]
@kousi2 - :+1: Looks good!

**Pull request merged**